system/hazard_status_converter/src/converter.cpp

```diff
@@ -148,7 +169,32 @@ void Converter::on_graph(const DiagnosticGraph::ConstSharedPtr msg)
     return HazardStatus::NO_FAULT;
   };
 
-  HazardStatusStamped hazard = convert_hazard_diagnostics(*msg);
+  const auto is_ignore = [this]() {
+    if (mode_) {
+      if (mode_->mode == OperationMode::AUTONOMOUS) {
```

Comment:
[Q] In [system_error_monitor](https://github.com/autowarefoundation/autoware.universe/blob/dfec62ac277f893fd6276e3c6ffd9c7d4f475225/system/system_error_monitor/src/system_error_monitor_core.cpp#L190) `GateMode` is used, but `OperationMode` is used here. What is the difference of these?

Reply:
It's almost the same. The operation mode is more abstract.
https://autowarefoundation.github.io/autoware.universe/main/system/default_ad_api/document/operation-mode/#compatibility

Reply:
FIxed for stop mode.
https://github.com/autowarefoundation/autoware.universe/pull/6733/commits/1e561c2994bf17589288e3519563b252525e02f5

Reply:
Thank you!